themes/jquery/js/modal.js

```diff
@@ -0,0 +1,53 @@
+// Store modal templates
+var modalTemplate = $( "#sri-modal-template" )[ 0 ].outerHTML,
+	clipboard;
+
+// Show modal on click
+$( "body" ).on( "click", ".open-sri-modal", function( event ) {
+	if ( !event.ctrlKey && !event.metaKey ) {
+		$( replace( modalTemplate, {
+			link: document.location.origin + $( this ).attr( "href" ),
+			hash: $( this ).attr( "data-hash" )
+		} ) ).removeAttr( "id" ).appendTo( "body" ).dialog( {
+			modal: true,
+			resizable: false,
+			width: 830,
+			dialogClass: "sri-modal",
+			draggable: false,
+			close: function() {
+				$( this ).remove();
+			}
+		} );
+
+		return false;
+	}
+} );
+
+// Helper functions
+function replace ( string, values ) {
+	$.each( values, function( key, value ) {
+		string = string.replace( new RegExp( "\\{\\{" + key + "}}", "g" ), value );
+	} );
```

Comment:
You can avoid generating `RegExp`s in a loop with a more generic template system:

```js
return string.replace((/\{\{([^}]+)}}/g, function(_, key) {
    return values[key];
});
```